recipes/rdflib/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "rdflib" %}
+{% set version = "4.2.1" %}
+{% set sha256 = "eb02bd235606ef3b26e213da3e576557a6392ce103efd8c6c8ff1e08321608c8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - isodate
+    - pyparsing
+    - html5lib
+    - sparqlwrapper
+
+  run:
+    - python
+    - setuptools
+    - isodate
+    - pyparsing
+    - html5lib
+    - sparqlwrapper
+
+test:
+  imports:
+    - rdflib
+    - rdflib.extras
+    - rdflib.plugins
+    - rdflib.plugins.parsers
+    - rdflib.plugins.serializers
+    - rdflib.plugins.sparql
+    - rdflib.plugins.stores
+    - rdflib.tools
+
+  commands:
```

Comment:
This block of `commands` should be under `build: entry_points`:

http://conda.pydata.org/docs/building/meta-yaml.html#python-entry-points

and then change the tests to something like:

```test:
   commands:
      - rdfpipe -h
      - csv2rdf -h
      ...
```

assuming that those are valid calls that return non-error responses.